tests/de.uka.ipd.sdq.workflow.mdsd.tests/src/de/uka/ipd/sdq/workflow/mdsd/tests/MDSDBlackboardTests.java

```diff
@@ -0,0 +1,95 @@
+package de.uka.ipd.sdq.workflow.mdsd.tests;
+
+
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import de.uka.ipd.sdq.workflow.jobs.CleanupFailedException;
+import de.uka.ipd.sdq.workflow.jobs.JobFailedException;
+import de.uka.ipd.sdq.workflow.jobs.UserCanceledException;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.MDSDBlackboard;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.ModelLocation;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.ResourceSetPartition;
+import de.uka.ipd.sdq.workflow.mdsd.blackboard.SavePartitionToDiskJob;
+import de.uka.ipd.sdq.workflow.mdsd.mocks.MockResourceSetPartition;
+
+import java.util.Map;
+
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+
+
+public class MDSDBlackboardTests {
+	
+	MDSDBlackboard blackboard;
+	
+	@BeforeClass
+	public static void registerFactories() {
+		Resource.Factory.Registry reg = Resource.Factory.Registry.INSTANCE;
```

Comment:
Please fix the indentation (in the whole file)

Reply:
new Eclipse --> wrong formatter